ui-v2/tests/unit/mixins/kv/with-actions-test.js

```diff
@@ -18,3 +24,29 @@ test('it works', function(assert) {
   const subject = this.subject();
   assert.ok(subject);
 });
+test('afterUpdate calls transitionTo index when the key is a single slash', function(assert) {
+  const subject = this.subject();
+  const expected = 'dc.kv.index';
+  const transitionTo = this.stub(subject, 'transitionTo').returnsArg(0);
+  const actual = subject.afterUpdate({}, { Key: '/' });
+  assert.equal(actual, expected);
+  assert.ok(transitionTo.calledOnce);
+});
+test('afterUpdate calls transitionTo folder when the key is not a single slash', function(assert) {
+  const subject = this.subject();
+  const expected = 'dc.kv.folder';
+  const transitionTo = this.stub(subject, 'transitionTo').returnsArg(0);
+  ['', '/key', 'key/name'].forEach(item => {
+    const actual = subject.afterUpdate({}, { Key: item });
+    assert.equal(actual, expected);
+  });
+  assert.ok(transitionTo.calledThrice);
+});
+test('afterDelete calls refresh folder when the routeName is `folder`', function(assert) {
+  const subject = this.subject();
+  subject.routeName = 'dc.kv.folder';
+  const refresh = this.stub(subject, 'refresh');
+  subject.afterDelete({}, {});
+  assert.ok(refresh.calledOnce);
+});
+skip('action invalidateSession test');
```

Comment:
Why is this skipped?

Reply:
Ah yeah so this functionality is not part of this PR/unit of work. If you look at the diff, this is pretty much unchanged. While I was doing this work, I did notice that this functionality isn’t unit tested (but is acceptance tested). So I added a `skip` here so it will forever bug me in my test output until I come back and sort this on a new branch/PR/unit of work.